src/urh/util/FileOperator.py

```diff
@@ -100,6 +100,7 @@ def get_save_file_name(initial_name: str, wav_only=False, caption="Save signal")
     if caption == "Save signal":
         name_filter = "Complex files (*.complex);;Complex16 files (2 unsigned int8) " \
                       "(*.complex16u);;Complex16 files (2 signed int8) (*.complex16s);;" \
+                      "CU8 files (*.cu8);;CS8 files (*.cs8);;" \
```

Comment:
It would be better to have (*.complex16u *.cu8) shared like in the open dialog instead of creating new entries.

Reply:
That wouldn't allow the user to select the desired extension then. Each entry there is one option in the format drop-down.

Reply:
I am not a fan of creating a new entry for file type which is essentially the same thing like ``` complex16u ``` (and s). You are right with your concern, that this would cause confusion. My idea was to simply show in the name filter it is possible to save as ``` *.cu8 ``` and, if you really want that, you just enter the extension manually.

Reply:
Nice!